src/controllers/hid/hidcontroller.cpp

```diff
@@ -298,6 +298,34 @@ void HidController::sendBytesReport(QByteArray data, unsigned int reportID) {
     }
 }
 
+void HidController::sendFeatureReport(
+        const QList<int>& dataList, unsigned int reportID) {
+    QByteArray dataArray;
+    for (const int datum: dataList) {
+        dataArray.append(datum);
+    }
+	
+    // Append the Report ID to the beginning of dataArray[] per the API..
+    dataArray.prepend(reportID);
```

Comment:
if you start by appending the reportID and then append the datalist, it'll be a little faster and easier to read

Reply:
In my opinion it's better, to have he same order of arguments for the mostly identical API methods controller.send and controller.sendFeatureReport
The controller.sendFeatureReport method will be rarely used (usually once in the .int section only), while the existing controller.send method will be used often accross the code. If performance is an issue, than controller.send needs to be changed.

Reply:
After a further comment from uklotzde I understood what you really meant here. Sorry, my first answer was false. I changed the code.